src/sections/ServicesTechnologies/index.css

```diff
@@ -0,0 +1,47 @@
+@import "../../styles/all.css";
+
+.root {
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.title > * {
+  text-align: center;
+}
+
+.technology-container {
+  display: flex;
```

Comment:
Maybe it's best to wrap this in a `tabletAndAbove` breakpoint, since you are altering the layout completely (change only things related to flexbox)